.travis.yml

```diff
@@ -4,6 +4,16 @@ php:
     - 7.1
     - 7.3
 
+matrix:
+    fast_finish: true
+
+    allow_failures: []
+
+    include:
+        - php: "7.3"
+          env:
+              - STANDALONE=1
```

Comment:
why is this needed?
this travis file is always standalone and will not be used in nonsplit mode etc.